extension.js

```diff
@@ -671,11 +711,12 @@ const EasyScreenCast_Indicator = new Lang.Class({
                     this.indicatorIcon.set_gicon(Lib.ESConGIcon);
                 }
             } else {
-                if (focus === true) {
+                this.indicatorIcon.set_gicon(null);
+/*                if (focus === true) {
```

Comment:
Remove the commented code?

Reply:
fixed issue: https://github.com/EasyScreenCast/EasyScreenCast/issues/155